wire-runtime/src/test/java/com/squareup/services/RxJavaService2SomeEndpoints.java

```diff
@@ -59,4 +42,18 @@ public RxJavaService2SomeEndpoints(Endpoint endpoint) {
   public Func1<LetsDataRequest, LetsDataResponse> getLetsData() {
     return letsData;
   }
+
+  public interface Endpoint {
+    /**
+     * Sends some more data.
+     */
+    @POST("/com.squareup.services.RxJavaService2/SendSomeMoreData")
```

Comment:
Note that this is the one behavior change. The annotation now retains the same name as the proto service unconditionally, even if a suffix is added to the generated interface.

Specifically the `SomeEndpoints` suffix is added to the interface but not to the annotation's value.
```
-    @POST("/com.squareup.services.RxJavaService2SomeEndpoints/LetsData")
-    LetsDataResponse letsData(@Body LetsDataRequest request);
```